scheduler/src/cook/mesos.clj

```diff
@@ -52,18 +52,18 @@
      (let [submit-time (java.util.Date.)]
        (doseq [{:keys [uuid command ncpus memory name retry-count priority]} jobs
                :let [txn {:db/id (d/tempid :db.part/user)
-                          :job/uuid uuid
```

Comment:
When adding a new element to the map literal, I think it's easier to add at the end than to have to remember (or be reminded) to add in the lexicographically sorted place. That combined with the fact that IDs are often "at the beginning" makes me question this.

At the end of the day, I don't feel that strongly about it, but I wonder if this could be causing more harm than good.